### DISCUSSION TITLE
Input folder filling up

### DISCUSSION THREAD

**Discussion by monogoat** [VorlonCD/bi-aidetection#277] [Q&A]
I'm running into a problem where the folder that BlueIris outputs alert images to for the AI Tool to scan is filling up with files. After a month or two BlueIris' alert list goes blank and each time I refresh the view any new alerts disappear. I can cure this by emptying the AI folder, which will have 15k+ images. 

The history in AI Tool is set for 14 days, but they are not deleted after 14 days. Is there some setting I don't have properly configured?

**Comment by VorlonCD** [comment]
AITOOL does not remove anything in the folder where BI creates files.   

You have to set BI itself to delete or archive the files via its Settings > Clips and archiving:   Set each folder to either be deleted after a certain age or size or move them to a folder on another drive for long term storage like I do.

**Reply by monogoat** [reply]
Ok, so BI is what isn't deleting them. The setting there doesn't seem to work, I have been shortening it (currently it is set at 2 days for the AI folder to delete and just emptied a months worth of images). I'll see if I can get some support from them.
